docs/overview/status/ProjectStatus.stories.mdx

```diff
@@ -0,0 +1,84 @@
+
+- [Versions](#versions)
+- [Packages](#packages)
+- [Components](#components)
+- [Supported browsers](#supported-browsers)
+
+## Versions <a id="versions" />
+
+Here you can find all supported UI Kit versions and their corresponding Design System and core dependencies versions.  
+The most recent version is recommended for use in production.
+
+| Release date   | UI Kit version                                                       | Design System version                                                                                     | React version | MUI version                |
+| -------------- | -------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------- | ------------- | -------------------------- |
+| March 2023     | [UI Kit v5.x](https://lumada-design.github.io/uikit/master)          | [Design System 5.x](https://designsystem.hitachivantara.com/)                                             | 17, 18        | [5.x](https://mui.com/)    |
+| September 2022 | [UI Kit v4.x](https://lumada-design.github.io/uikit/v4)              | [Design System 3.x](https://hitachivantara.sharepoint.com/sites/DesignSystem/Pattern%20Library/Home.aspx) | 16, 17        | [5.x](https://mui.com/)    |
+| October 2020   | [UI Kit v3.x](https://lumada-design.github.io/uikit/v3-old-registry) | [Design System 3.x](https://hitachivantara.sharepoint.com/sites/DesignSystem/Pattern%20Library/Home.aspx) | 16, 17        | [4.x](https://v4.mui.com/) |
+
+To know more about the practices that are followed for release and versioning, please read our [**release and versioning**](/docs/overview-community-releases--page) documentation.
+
+## Packages <a id="packages" />
+
+At the moment, UI Kit v5.x has not available all v4.x packages. We are actively working toward aligning the missing packages with v5.x and they will be released as soon as they are ready, following our [**release cycle**](/docs/overview-community-releases--page) rules.
+
+Bear in mind that v4.x and v5.x packages can't be used simultaneously due to the introduction of the new theming system not compatible with the v4.x components.
+
+| Package                                 |    Status     |
+| --------------------------------------- | :-----------: |
+| @hitachivantara/uikit-react-core        |      ✅       |
+| @hitachivantara/uikit-react-icons       |      ✅       |
+| @hitachivantara/uikit-react-shared      |      ✅       |
+| @hitachivantara/uikit-styles            |      ✅       |
+| @hitachivantara/uikit-react-lab         | `in progress` |
+| @hitachivantara/uikit-react-viz         | `in progress` |
+| @hitachivantara/uikit-react-code-editor | `in progress` |
+| @hitachivantara/uikit-react-compat      | `deprecated`  |
+| @hitachivantara/uikit-common-themes     | `deprecated`  |
+
+## Components <a id="components" />
```

Comment:
The following components are missing from the components table: date picker, slider, time picker, and scroll to (horizontal and vertical)